From da821f36d7f0033a4d85bfa1a7450bc01981e43e Mon Sep 17 00:00:00 2001
From: William Wu <william.wu@rock-chips.com>
Date: Fri, 20 Nov 2020 15:53:39 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3568: delete the u2 susphy quirk for
 dwc3

This patch delete the "snps,dis_u2_susphy_quirk" property for
dwc3 controllers. So that the USB2.0 PHY can enter suspend mode
if the suspend conditions are valid.

Change-Id: I256a258016d1931adc58b3e49ea828575c500c58
Signed-off-by: William Wu <william.wu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3568.dtsi | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3568.dtsi b/arch/arm64/boot/dts/rockchip/rk3568.dtsi
index f3335cb621f2..2865e8946c29 100644
--- a/arch/arm64/boot/dts/rockchip/rk3568.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3568.dtsi
@@ -305,7 +305,6 @@
 			reset-names = "usb3-otg";
 			snps,dis_enblslpm_quirk;
 			snps,dis-u2-freeclk-exists-quirk;
-			snps,dis_u2_susphy_quirk;
 			snps,dis-del-phy-power-chg-quirk;
 			snps,dis-tx-ipgap-linecheck-quirk;
 			snps,xhci-trb-ent-quirk;
@@ -337,7 +336,6 @@
 			reset-names = "usb3-host";
 			snps,dis_enblslpm_quirk;
 			snps,dis-u2-freeclk-exists-quirk;
-			snps,dis_u2_susphy_quirk;
 			snps,dis-del-phy-power-chg-quirk;
 			snps,dis-tx-ipgap-linecheck-quirk;
 			snps,xhci-trb-ent-quirk;
-- 
2.35.3

